build-images.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-docker build --file 16.13/Dockerfile -t cimg/node:16.13.1  -t cimg/node:16.13  -t cimg/node:lts .
-docker build --file 16.13/browsers/Dockerfile -t cimg/node:16.13.1-browsers  -t cimg/node:16.13-browsers  -t cimg/node:lts-browsers .
+docker build --file 12.22/Dockerfile -t cimg/node:12.22.8  -t cimg/node:12.22  -t cimg/node:lts .
+docker build --file 12.22/browsers/Dockerfile -t cimg/node:12.22.8-browsers  -t cimg/node:12.22-browsers  -t cimg/node:lts-browsers .
```

Comment:
@JalexChen @felicianotech Due to this change `cimg/node:lts` now contains not Node.js v12 instead of v16, causing our build jobs failing.

Reply:
This was fixed about 20 minutes ago. Thanks.